ast/ast.go

```diff
@@ -1130,6 +1131,38 @@ type Duration struct {
 	Unit      string `json:"unit"`
 }
 
+func (l *Duration) Duration() (time.Duration, error) {
```

Comment:
I'm hesitant to put this in here as an API with an immediate deprecation notice, but the todo implies this function is deprecated without an actual godoc saying it is deprecated. If this is needed for a temporary implementation, would copy/paste be more efficient and less likely to result in a breaking api change?

Reply:
The TODO existed in the previous code, wether the code lives here or elsewhere it needs to be refactored in the future. Having only one copy means we can refactor all its uses at once.

Reply:
My main point was that previously it was on an unexposed function. But, I think that it is fine. I think that this is the meat of the PR anyway. I would like this to have a godoc that notes that the `time.Duration` is an approximation and does not handle timezones so that a person knows that this conversion is lossy.